src/vs/workbench/parts/terminal/electron-browser/terminalFindWidget.ts

```diff
@@ -44,4 +47,12 @@ export class TerminalFindWidget extends SimpleFindWidget {
 	protected onFocusTrackerBlur() {
 		this._terminalService.getActiveInstance().notifyFindWidgetFocusChanged(false);
 	}
+
+	protected onFindInputFocusTrackerFocus() {
```

Comment:
We'd better not expose input box to external callers, let's handle input, checkboxes inside the widget.